app/assets/stylesheets/profile.scss

```diff
@@ -0,0 +1,14 @@
+.edit-profile-page-astrikstyle {
```

Comment:
import this file from application.scss

Reply:
@tachyons  can you elaborate i can't understand what to import please

Reply:
![Screenshot (76)](https://user-images.githubusercontent.com/49101492/75653431-97794600-5c83-11ea-9001-4df88c3f9d99.png)
 like this?? @tachyons